notebooks/community/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -0,0 +1,1272 @@
+{
```

Comment:
<p>df = bq_query(...)</p><p>print(df)</p><p> </p><p>Move <code>For more information on how this data was prepared, read this blog post. </code> to the end of the cell. Rephrase as:</p><p> </p><p>Learn more about [ how this data was prepared](blog link)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>